UI/message.ts

```diff
@@ -58,6 +61,29 @@ function* match(regex: RegExp, content: string, type: ItemType): Generator<Conte
                     end: urlEndPosition,
                 };
             }
+        } else if (type == "nprofile") {
+            let bech32: string;
+            if (match[0].startsWith("nostr:")) {
+                bech32 = content.slice(urlStartPosition + 6, urlEndPosition + 1);
+            } else {
+                bech32 = content.slice(urlStartPosition, urlEndPosition + 1);
+            }
+            const decoded_nProfile = NostrProfile.decode(bech32);
+            if (decoded_nProfile instanceof Error) {
+                // ignore
+            } else {
+                const pubkey = PublicKey.FromBech32(decoded_nProfile.pubkey.hex);
```

Comment:
should be PublicKey.FromBech32(decoded_nProfile.pubkey.bech32());

Reply:
or PublicKey.FromHex(decoded_nProfile.pubkey.hex);

Reply:
Should be `decoded_nProfile.pubkey`

Reply:
ummm, lol